package/yast2-packager.spec

```diff
@@ -62,6 +65,9 @@ Requires:       /usr/bin/md5sum
 # .process agent
 Requires:       yast2-core >= 2.16.35
 
+# Storage (methods to check the devicegraph)
+Requires:       libstorage-ng-ruby
```

Comment:
It's a library, so the very idea is to have it general enough so it's useful without any glue code.

The glue code will no doubt make it easier to use, but it should not be absolutely necessary to use any glue code.